src/messages.json

```diff
@@ -147,7 +147,7 @@
         "KeyboardInputModal.Description.increaseProgramSpeed": "Press {key} to make the program play faster.",
         "KeyboardInputModal.Description.playPauseProgram": "Press {key} to play or pause the program.",
         "KeyboardInputModal.Description.refreshScene": "Press {key} to refresh the scene.",
-        "KeyboardInputModal.Description.showHide": "Press {key} to show / hide the keyboard shortcuts menu.",
+        "KeyboardInputModal.Description.showHide": "Press {key} to show the keyboard shortcuts menu.",
```

Comment:
Fair point, that used to be the case, I think?

Reply:
Either way, good catch.

Reply:
Thanks @the-t-in-rtf. I've switched to the icons that Sepideh gave me based on the conversation we had. I've also changed the filenames to be more understandable.